Userland/cal.cpp

```diff
@@ -98,6 +98,70 @@ void clean_buffers()
     temp_buffer[line_width - 1] = '\0';
 }
 
+void append_to_print(char* buffer, int row, int column, char* text)
+{
+    int starting_point = (line_width * row) + (column * column_width);
+    for (int i = 0; text[i] != '\0'; i++) {
+        buffer[starting_point + i] = text[i];
+    }
```

Comment:
This loop can be replaced with a `strlen()` + `memcpy()` invocation. I'm not sure if it _should_ be, because it's kind of more clear this way?

Reply:
I honestly don't really know how is the best way. However if you think this is more clear I'll let it be like it is for now.

However those functions probably are more efficient and fail-proof.